src/controller/osw-controller.ts

```diff
@@ -144,6 +144,7 @@ class OSWController implements IController {
         // Route for quality metric request
         this.router.post(`${this.path}/quality-metric/:tdei_dataset_id`, qualityUpload.single('file'),authenticate, this.createQualityOnDemandRequest);
         this.router.post(`${this.path}/quality-metric/tag/:tdei_dataset_id`, tagQuality.single('file'), authenticate, this.tagQualityMetric);
+        this.router.post(`${this.path}/dataset-add-inclination/`, authenticate, this.createInclineRequest);
```

Comment:
rename dataset-add-inclination => dataset-inclination . POST suggests it is add.

Reply:
Remove trailing / from url

Reply:
If dataset_id is mandatory and inclination is for dataset_id , then accept the input as part of path variable